src/signals/incident-management/containers/DefaultTextsAdmin/index.js

```diff
@@ -69,9 +72,23 @@ const DefaultTextsAdmin = ({
   </Fragment>
 );
 
+DefaultTextsAdmin.defaultProps = {
+  defaultTextsAdmin: {
+    defaultTexts: [],
+    defaultTextsOptionList: [],
+    categoryUrl: 'bla',
```

Comment:
Should this prop have the value 'bla' as default?

Reply:
gedaan